provisioning/service/src/Twin/ProvisioningTwinState.cs

```diff
@@ -9,10 +9,10 @@ namespace Microsoft.Azure.Devices.Provisioning.Service
     /// Representation of a single Twin initial state for the Device Provisioning Service.
     /// </summary>
     /// <remarks>
-    /// The TwinState can contain one <see cref="TwinCollection"/> of Tags, and one
-    /// <see cref="TwinCollection"/> of properties.desired.
+    /// The TwinState can contain one <see cref="ProvisioningTwinProperties"/> of Tags, and one
+    /// <see cref="ProvisioningTwinProperties"/> of properties.desired.
```

Comment:
should `properties.desired` have some kind of reference link? It isn't clear to me what that means.